packages/caliper-fabric/test/connector-versions/v1/WalletFacade.js

```diff
@@ -0,0 +1,141 @@
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+chai.should();
+const mockery = require('mockery');
+
+/**
+ * simulate a node sdk v1 wallet
+ */
+class StubWallet {
+    /**
+     * Mock V1 wallet implementation
+     */
+    constructor() {
+        this.map = new Map();
+    }
+
+    /**
+     *
+     * @param {*} key b
+     * @param {*} value b
+     */
+    async import(key, value) {
+        this.map.set(key, value);
+    }
+
+    /**
+     *
+     * @param {*} key b
+     */
+    async export(key) {
+        return this.map.get(key);
+    }
+
+    /**
+     *
+     */
+    async list() {
+        return Array.from(this.map.keys());
+    }
+}
+
+/**
+ * InMemoryWallet copy
+ */
+class InMemoryWallet extends StubWallet {}
+
+/**
+ * FileSystemWallet copy
+ */
+class FileSystemWallet extends StubWallet {}
+
+/**
+ * x509walletmixin class
+ */
+class X509WalletMixin {
+    /**
+    * @param {*} mspid b
+    * @param {*} cert b
+    * @param {*} key b
+    * @return {*} identity b
+    */
+    static createIdentity(mspid, cert, key){
+        const identity = {
+            credentials: {
+                cert,
+                key
+            },
+            mspid,
+            type: 'X.509',
+        };
+        return identity;
+    }
+}
+
+mockery.enable();
+mockery.registerMock('fabric-network',  {FileSystemWallet, InMemoryWallet, X509WalletMixin});
+
+const WalletFacadeFactory = require('../../../lib/connector-versions/v1/WalletFacadeFactory');
+const WalletFacade = require('../../../lib/connector-versions/v1/WalletFacade');
+
+describe('When testing a V1 Wallet Facade Implementation', () => {
+
+    beforeEach(() => {
+        mockery.enable();
+        mockery.registerMock('fabric-network',  {FileSystemWallet, InMemoryWallet, X509WalletMixin});
+    });
+
+    afterEach(() => {
+        mockery.deregisterAll();
+        mockery.disable();
+    });
+    it('A Wallet Facade Factory should create a wallet facade', async () => {
+        const walletFacade = await new WalletFacadeFactory().create();
+        walletFacade.should.be.instanceOf(WalletFacade);
+        const walletFacade2 = await new WalletFacadeFactory().create('optionalString');
+        walletFacade2.should.be.instanceOf(WalletFacade);
+    });
+
+    it('A wallet facade should be able to import and export identities', async () => {
+        const walletFacade = await new WalletFacadeFactory().create();
+        await walletFacade.import('mspid', 'label', 'cert', 'key');
+        const exported = await walletFacade.export('label');
+        exported.should.deep.equal({mspid: 'mspid', certificate: 'cert', privateKey: 'key'});
+    });
+
```

Comment:
I'm going to add this test in as well to the v2 tests so could you add this extra test in here as well
```javascript
    it('A wallet facade should return null on export if the identity does not exist', async () => {
        const walletFacade = await new WalletFacadeFactory().create();
        const exported = await walletFacade.export('label');
        should.equal(exported, null);
    });
```